.evergreen/evergreen.yml

```diff
@@ -134,12 +134,13 @@ buildvariants:
 
   - name: generator
     display_name: Generator
-    allowed_requesters: ["github_pr"]
+    patch_only: true
```

Comment:
I was curious about the decision to limit this to patch builds—do you think it might be helpful to extend it to mainline commits too? It could potentially solve the issue of missing task statuses on base commits for [patches](https://spruce.mongodb.com/task/evergreen_ui_spruce_display_e2e_parallel_patch_7a69ecadec70775e06d1dd8b01793117ad0824ff_66f30af220f57e0007be7617_24_09_24_18_54_43?execution=0&sortBy=STATUS&sortDir=ASC).

Reply:
That's an option, but it would require more changes to the YAML and I don't want to overload this PR. I'm pretty sure that if I uncommented `patch_only` and merged the PR as-is, it would break the builds on the waterfall. I think we can reevaluate this a later time